### ISSUE TITLE
HTTP Healthcheck endpoint

### ISSUE THREAD

**Issue by steve-thousand** [itzg/docker-minecraft-server#1118]
### Enhancement Type

A completely new feature

### Describe the enhancement

I would like to run the server in AWS. Unlike the guide listed in the readme, I would like to put a Fargate task behind a load balancer. This requires specifying a simple health-check path/port/protocol for a target group.

Minecraft's particular protocol doesn't seem to play well with target groups.

**Comment by steve-thousand** [comment]
Update on this: it turns out I was using the wrong kind of balancer. I was using an application load balancer which wouldn't allow plain TCP traffic, only HTTP. Once I switched to a network load balancer, I was able to get target groups to register healthy tasks with plain TCP protocol on port 25565.

Given that this might have just been a user-error on my part, this request might not be necessary to fulfill. It would still be useful to have a status endpoint, or the option to turn one on.
